### PR TITLE
STRF-7393 bump paper-handlebars version to 4.2.2

### PR DESCRIPTION
**Pull request by bc-williamkwon**
This pr is part of a series of prs to upgrade cli to paper 3.x branch.

https://jira.bigcommerce.com/browse/STRF-7393

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 3.0.0-rc.21 (2019-10-02)
+- Fix renderTheme function in order to not lose templates in stencil cli [#179](https://github.com/bigcommerce/paper/pull/179)
```
Comment (junedkazi) [inline CHANGELOG.md]: Can we also add there is bump for paper handlebars

File: index.js [hunk 2]
```diff
@@ -250,8 +250,8 @@ class Paper {
             // compile all the template required files into an object
             result = {};
             for (let path in templatePath) {
-                renderPromises.push(this.render(path, data.context).then(html => {
-                    result[path] = html;
+                renderPromises.push(this.render(templatePath[path], data.context).then(html => {
```
Comment (junedkazi) [inline index.js]: How did this even work before ?

Comment (junedkazi) [reply index.js]: Can we get some tests around it ?

File: CHANGELOG.md (hunk 1 above)
Comment (bc-williamkwon) [reply CHANGELOG.md]: fixed.

File: index.js (hunk 2 above)
Comment (bc-williamkwon) [reply index.js]: This was never used before.  Ok.

Comment (junedkazi) [inline index.js]: @mattolson can you take a look at this change.

**Pull request merged**